### DISCUSSION TITLE
Some feature this app still miss.

### DISCUSSION THREAD

**Discussion by lessload** [celzero/rethink-app#844] [General]
Main feture that firewall app consider to have it.
- Order manager of ip rule like nftables. to make user know which rule apply first

Some feature that still miss on android.
- Not show app name of app in "Work profile" or other profile, in Xioami it's call "Dual apps"

Question
- Is it posible to use one config file with multiple device ? 
  App rule still stay in config ? If another device doesn't have that app. ( one config for multiple device is impossible if the rule not store on all device )

**Comment by ignoramous** [comment]
Thanks.

> Order manager of ip rule like nftables. to make user know which rule apply first

This is a problem. We should figure out a way to show this: https://github.com/celzero/rethink-app/issues/781 

> Not show app name of app in "Work profile" or other profile, in Xioami it's call "Dual apps"

This is a restriction in Android. Rethink can't really "see" *Dual Apps* (because they run in a separate "profile"?) except for their IDs.

> Is it posible to use one config file with multiple device? App rule still stay in config?

Not yet, but planned: #757
